website/docs/docs/develop/dbt-cloud-tips.md

```diff
@@ -0,0 +1,73 @@
+- The [`dbt_project_evaluator`](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest) package compares your dbt project against a list of our best practices and provides suggestions and guidelines on how to update your models.
+- The [`dbt_expectations`](https://hub.getdbt.com/calogica/dbt_expectations/latest) package contains many tests beyond those built into dbt Core.
+- The [`dbt_audit_helper`](https://hub.getdbt.com/#:~:text=adwords-,audit_helper,-codegen) package lets you compare the output of 2 queries. Use it when refactoring existing logic to ensure that the new results are identical.
+- The [`dbt_artifacts`](https://hub.getdbt.com/brooklyn-data/dbt_artifacts/latest) package saves information about your dbt runs directly to your data platform so that you can track the performance of models over time.
+- The [`dbt_meta_testing`](https://hub.getdbt.com/tnightengale/dbt_meta_testing/latest) package checks that your dbt project is sufficiently tested and documented.
+
+## Advanced tips
+
+- Use your folder structure as your primary selector method. `dbt build marts.marketing` is simpler and more resilient than relying on tagging every model.
+- Think about jobs in terms of build cadences and SLAs. Run models that have hourly, daily, or weekly build cadences together.
+- Use the [`where` config](/docs/reference/resource-configs/where) for tests to test an assertion on a subset of records.
+- [`store_failures`](/docs/reference/resource-configs/store_failures) lets you examine records that cause tests to fail, so you can either repair the data or change the test as needed.
+- Use [`severity`](/docs/reference/resource-configs/severity) thresholds to set an acceptable number of failures for a test.
+- Use [`incremental_strategy`](/docs/building-a-dbt-project/building-models/configuring-incremental-models#configuring-incremental-strategy) in your incremental model config to implement the most effective behavior depending on the volume of your data and reliability of your unique keys.
+- Set `vars` in your `dbt_project.yml` to define global defaults for certain conditions, which you can then override using the `--vars` flag in your commands.
+- Use [`for loops`](/docs/guides/getting-started/learning-more/using-jinja#use-a-for-loop-in-models-for-repeated-sql) in Jinja to [DRY](/docs.getdbt.com/terms/dry) up repetitive logic, such as selecting a series of columns that all require the same transformations and naming patterns to be applied.
+- Instead of relying on post-hooks, use the [`grants` config](/docs/reference/resource-configs/grants) to apply permission grants in the warehouse resiliently.
+- Define [`source-freshness`](/docs/building-a-dbt-project/using-sources#snapshotting-source-data-freshness) thresholds on your sources to avoid running transformations on data that has already been processed.
```

Comment:
```suggestion
- Define [source-freshness](/docs/building-a-dbt-project/using-sources#snapshotting-source-data-freshness) thresholds on your sources to avoid running transformations on data that has already been processed.
```